torch/testing/_core.py

```diff
@@ -275,6 +276,112 @@ def make_non_contiguous(tensor: torch.Tensor) -> torch.Tensor:
     return input.data
 
 
+def make_tensor(size, device: torch.device, dtype: torch.dtype, *, low=None, high=None,
```

Comment:
1. IMHO nothing besides `size` should be a positional parameter. 

```suggestion
def make_tensor(size, *, device: torch.device, dtype: torch.dtype, low=None, high=None,
```

Since this diverges from our internal signature, I suggest doing to following in `torch.testing._internal.common_utils`

```python
from torch.testing import make_tensor as _make_tensor

def make_tensor(size, device, dtype, **kwargs):
    return _make_tensor(size, device=device, dtype=dtype, **kwargs)
```

2. For the public version, `device` and `dtype` should have default values. I suggest we use `"cpu"` and `torch.float32`.

Reply:
Thanks for the suggestion, @pmeier. I think it's a good idea to make them keyword-only since we are now exposing it to the public API. I have made the changes.